scripts/mytop.sh

```diff
@@ -285,25 +295,28 @@ ReadMode($RM_RESET) unless $config{batchmode};
 
 my $db_version;
 my $db_release;
-my $server="MySQL";
+my $server = "MySQL";
 my $have_query_cache;
 
-my @variables = Hashes("show variables");
+my @variables = Hashes("SHOW VARIABLES");
 
 foreach (@variables)
 {
     if ($_->{Variable_name} eq "version")
     {
         $db_version = $_->{Value};
-	$db_version =~ /(\d+)/;
-        $db_release= $1;
-        $server="MariaDB" if ($db_version =~ /maria/i);
+	$db_version =~ /^(\d+)/;
+        $db_release = $1;
+        $server = "MariaDB" if ($db_version =~ /maria/i);
+        if($db_version =~ m/(.*?)-/)
```

Comment:
I would suggest here construction as:
```
+        # Get the version number only
+        $db_version = $1 if($db_version =~ m/(.*?)-/);
```
And also note that we could add here information about the build type: `10.2.4-debug`